### PR TITLE
Te2116 certification tag xss

### PR DESCRIPTION
**Pull request by caxter99**

### CODE REVIEW

**Comment by jkulton** [comment]
whomst

File: build/merged_templates.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+(function(module) {
```
Comment (DougKeller) [inline build/merged_templates.js]: these build files are temporary and can be removed with `gulp clean`

**Review by DougKeller — CHANGES_REQUESTED** [review]

**Comment by coreyconley** [comment]
> whomst

Lol this bad boy is getting a lot of mileage.

**Comment by DougKeller** [comment]
> Lol this bad boy is getting a lot of mileage.

angucomplete-alt will never die :joy: always nice to see you guys stop by @jkulton @coreyconley

**Review by DougKeller — APPROVED** [review]

**Pull request merged**